src/idsk/components/search-results/search-results.js

```diff
@@ -0,0 +1,686 @@
+import '../../../govuk/vendor/polyfills/Function/prototype/bind'
```

Comment:
(* plati v podstate pre cely tento folder) celkovo si nemyslim zeby toto malo byt medzi komponentami, malo by to byt vo vzoroch, komponenty by mali byt len filtre 3 typy (radio, datum, checkbox).. kedze ale termin bol nastaveny nie podla casu kolko trva vyvoj ale bol zrazu dany, musim akceptovat toto. V ramci update-u by som to ale neskor premiestnil tam kam to patri